docs/api/search/pif/query/core/PropertyQuery.md

```diff
@@ -0,0 +1,13 @@
+## Description
+
+A PropertyQuery object contains information about a desired property.
+
+## Fields
+
+Field name | Value type | Default | Description
+-----------|------------|---------|------------
+`name` | [`FieldOperation`](api/search/pif/query/core/FieldOperation) |  | Acceptable names of the property as a field operation |
```

Comment:
We should have a better description for fields that use a FieldOperation. Something like "Operations to execute against the name field." It doesn't necessarily need to be a filter, but the current description implies that.